src/api/demo.rkt

```diff
@@ -338,16 +338,7 @@
    (url main)))
 
 (define (check-status req job-id)
-  (define r (get-results-for job-id))
-  ;; TODO return the current status from the jobs timeline
-  (match r
-    [#f
-     (response 202
-               #"Job in progress"
-               (current-seconds)
-               #"text/plain"
-               (list (header #"X-Job-Count" (string->bytes/utf-8 (~a (job-count)))))
-               (λ (out) (display "Not done!" out)))]
+  (match (get-timeline-for job-id)
     [(? box? timeline)
```

Comment:
What's the point in making this a `box`? Like, it won't be the same box as in the worker thread, so why not just ship back the box contents instead?